internal/server/presentation/log.go

```diff
@@ -19,6 +19,11 @@ type AMQPLogHandler struct {
 	Channel    *amqp.Channel
```

Comment:
Looking at the Front implementation, it follows a standard work queue pattern.

https://www.rabbitmq.com/tutorials/tutorial-two-go

However, this code adopts an RPC system, which results in some unnecessary code.

https://www.rabbitmq.com/tutorials/tutorial-six-go

Since it seems more appropriate to align with the work queue pattern in this case, the implementation needs to be modified.